src/templates/registration/logged_out.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'base.html' %}
+{% load static %}
+
+<link rel="stylesheet" href="{% static "css/auth.css" %}">
```

Comment:
we should be including this via some kind of `{% block "extra_headers" %}` that is in the `base.html` `<head>` tag